scripts/rhel_validation/prepare_lis_vm.sh

```diff
@@ -48,9 +49,13 @@ function prepare_vm {
 
     popd
 
-    wget https://aka.ms/downloadazcopy-v10-linux
-    tar xvf downloadazcopy-v10-linux
-    ./azcopy_linux_amd64_10.1.2/azcopy cp "${kernel_version}" "https://${storage_account}.blob.core.windows.net/kernel/${storage_sas_token}" --recursive
+    storage_account=$(echo $storage_account|sed -e 's/[\r\n]//g')
+    storage_sas_token=$(echo $storage_sas_token|sed -e 's/[\r\n]//g')
+    azcopy_downloadlink=$(echo $azcopy_downloadlink|sed -e 's/[\r\n]//g')
+
+    wget -O azcopy.tar.gz ${azcopy_downloadlink}
+    tar xvf azcopy.tar.gz
```

Comment:
remove verbose flag, not needed in automation

Reply:
removed verbose flag